### PR TITLE
Add changelog for alpine super and fix version reporting

### PR DESCRIPTION
**Pull request by pcarranzav**
connects to #147

---
This change is [Reviewable](https://reviewable.io/reviews/resin-io/resin-supervisor/148)

### CODE REVIEW

**Comment by pcarranzav** [comment]
Currently we report the version from package.json to show on the dashboard. This PR adds an `-alpine` to the tag using an env var in the Dockerfile, so that it is correctly shown in the dashboard as e.g. "1.8.0-alpine". Will be especially important if we decide to allow users to choose supervisor versions with a dropdown or the like.

File: src/utils.coffee [hunk 1]
```diff
@@ -14,7 +14,10 @@ logger = require './lib/logger'
 utils = exports
 
 # Parses package.json and returns resin-supervisor's version
-exports.supervisorVersion = require('../package.json').version
+version = require('../package.json').version
+tagExtra = process.env.SUPERVISOR_TAG_EXTRA
+version += '-' + tagExtra if !_.isEmpty(tagExtra)
```
Comment (Page-) [inline src/utils.coffee]: Can you use a `+` please, to avoid issues with `'x-y' < 'x'`

**Comment by Page-** [comment]
I personally would be all behind just having alpine as the supervisor base image to use, but I still think this is potentially useful if people want to roll a custom supervisor

**Comment by lekkas** [comment]
Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[src/utils.coffee, line 19 \[r1\]](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148#-KI4E65DekDWJAxFQiB7-r1-19:-KI7HqeuFf7rSxST7Qti:5172523) ([raw file](https://github.com/resin-io/resin-supervisor/blob/99ecd92cc4b123f933a9ed277d273c367e6b8933/src/utils.coffee#L19)):*
<details><summary><i>Previously, Page- wrote…</i></summary>
> Can you use a `+` please, to avoid issues with `'x-y' < 'x'`
</details>
+1

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148)*

**Comment by pcarranzav** [comment]
Yeah, let's leave that for a bit later after giving alpine some more thorough testing.

<details><summary><i>Previously, Page- wrote…</i></summary>
> I personally would be all behind just having alpine as the supervisor base image to use, but I still think this is potentially useful if people want to roll a custom supervisor
</details>


---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[src/utils.coffee, line 19 \[r1\]](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148#-KI4E65DekDWJAxFQiB7-r1-19:-KI87bBCvTcnFQlSGUvt:729018781) ([raw file](https://github.com/resin-io/resin-supervisor/blob/99ecd92cc4b123f933a9ed277d273c367e6b8933/src/utils.coffee#L19)):*
<details><summary><i>Previously, lekkas (Kostas Lekkas) wrote…</i></summary>
> +1
</details>
Agreed, but do we leave the image tagged with a '-'?  Cause '+' is not valid for a docker image tag.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148#-:-KI88vJfR9Mg3IgSDBGU:-994680286)*

**Comment by lekkas** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[src/utils.coffee, line 19 \[r1\]](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148#-KI4E65DekDWJAxFQiB7-r1-19:-KI8RHeBwUJxIX2bGonK:1646537978) ([raw file](https://github.com/resin-io/resin-supervisor/blob/99ecd92cc4b123f933a9ed277d273c367e6b8933/src/utils.coffee#L19)):*
<details><summary><i>Previously, pcarranzav (Pablo Carranza Vélez) wrote…</i></summary>
> Agreed, but do we leave the image tagged with a '-'?  Cause '+' is not valid for a docker image tag.
</details>
Hm, docker has [an pending/open PR](https://github.com/docker/distribution/pull/1202) for semver-compatible tags btw. 

I think it's ok to leave the '-' in the tagged image and use '+' in the supervisor version to keep `alpine` as semver-compatible metadata. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148)*

**Comment by pcarranzav** [comment]
@Page- @lekkas fixed to report with a '+'.

**Comment by lekkas** [comment]
:lgtm:

<details><summary><i>Previously, pcarranzav (Pablo Carranza Vélez) wrote…</i></summary>
> @Page- @lekkas fixed to report with a '+'.
</details>


---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/resin-io/resin-supervisor/148#-:-KI8rwA0NVq8vD836CXq:1417088721)*

**Pull request merged**